packages/extension-polkagate/src/popup/staking/pool/stake/createPool/index.tsx

```
@@ -154,11 +154,11 @@
     }
 
     if (!api?.call?.['transactionPaymentApi']) {
-      return setEstimatedFee(api.createType('Balance', BN_ONE));
+      return setEstimatedFee(api.createType('Balance', BN_ONE) as Balance);
     }
 
     api && api.tx['nominationPools']['create'](String(amountAsBN.gte(BN_ONE) ? amountAsBN : BN_ONE), formatted, nominatorId, bouncerId).paymentInfo(formatted).then((i) => {
-      setEstimatedFee(api.createType('Balance', i?.partialFee));
+      setEstimatedFee(api.createType('Balance', i?.partialFee) as Balance);
     }).catch(console.error);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)